packages/server/customer-os-api/repository/contact_repository.go

```diff
@@ -717,7 +717,26 @@ func (r *contactRepository) MergeContactRelationsInTx(ctx context.Context, tx ne
 		"			SET	rel.merged=true", params); err != nil {
 		return err
 	}
-
+	if _, err := tx.Run(ctx, matchQuery+
+		" WITH primary, merged "+
+		" MATCH (merged)<-[rel:CREATED_BY]-(m:Meeting) "+
+		" MERGE (primary)<-[newRel:CREATED_BY]-(m) "+
+		" ON CREATE SET newRel.mergedFrom = $mergedContactId, "+
+		"				newRel.createdAt = $now, "+
+		"				newRel.type = rel.type "+
+		"			SET	rel.merged=true", params); err != nil {
+		return err
+	}
+	if _, err := tx.Run(ctx, matchQuery+
+		" WITH primary, merged "+
+		" MATCH (merged)<-[rel:ATTENDED_BY]-(m:Meeting) "+
+		" MERGE (primary)<-[newRel:ATTENDED_BY]-(m) "+
+		" ON CREATE SET newRel.mergedFrom = $mergedContactId, "+
+		"				newRel.createdAt = $now, "+
+		"				newRel.type = rel.type "+
```

Comment:
is type field belongs to property has ATTENDED_BY / CREATED_BY ?
if not to be excluded.

Reply:
type property from relation is going to be taken out by me in a new PR ( tomorrow ). I will do the changes for this on my side